package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "seneca-transport",
-  "version": "0.6.5",
+  "version": "0.6.6",
```

can we postfix forks so we can confuse with source versions. ie.:
```suggestion
  "version": "0.6.6-caseiq",
```